### DISCUSSION TITLE
Libressl port for Harvey

### DISCUSSION THREAD

**Discussion by bugcrazy** [Harvey-OS/harvey#1004] [Ideas]
Libressl was ported to Plan 9 libressl-portable/portable#510, I believe it will be very useful for the Harvey project, also has ASM for different architectures here  [libressl-portable-asm](https://github.com/q66/libressl-portable-asm).

**Comment by elbing** [comment]
It needs APE or APEX (unfinished and buggy) in case of Harvey. So it's not a native port. About portable-asm, Harvey is intended to not use assembly code except for those cases in what it's absolutely imperative. It's problematic and hard of being maintained.
Reality is that there's no ports projects in Harvey being actively developed, and this is useless since we can't make other previous needed ports like GCC and many others, and there's nothing into the main system which needs for now something like that. Libressl is not a priority in actual status.
Thanks for the info, anyway.
IMHO this should be closed.
